tap_mambu/tap_mambu_refactor/tap_generators/branches_generator.py

```diff
@@ -6,9 +6,5 @@ def _init_endpoint_config(self):
         super(BranchesGenerator, self)._init_endpoint_config()
         self.endpoint_path = "branches"
         self.endpoint_api_method = "GET"
-        self.endpoint_params = {
-            "sortBy": "lastModifiedDate:ASC",
-            "detailsLevel": "FULL",
```

Comment:
Does the removal of `FULL` here change the data returned or is it not applicable to this stream?